examples/two_dimensional_turbulence.jl

```diff
@@ -0,0 +1,58 @@
+# # Two dimensional turbulence example
+#
+# In this example, we initialize a random velocity field and observe its viscous,
+# turbulent decay in a two-dimensional domain.
+
+using Oceananigans, PyPlot, Statistics
+
+# In addition to importing plotting and statistics packages, we import
+# some types and functions from `Oceananigans` that will aid in the calculation
+# and visualization of voriticty.
+
+using Oceananigans: Face, Cell
+using Oceananigans.TurbulenceClosures: ∂x_faa, ∂y_afa
```

Comment:
Hmmm, I feel like examples shouldn't be importing unexported functions.

Reply:
What's wrong with unexported functions?

Reply:
@ali-ramadhan thoughts?